pipelines/prep_data.py

```diff
@@ -609,11 +649,12 @@ def process_and_save_loc_param(
         else:
             pop_fraction = subpop_sizes / sum(subpop_sizes)
 
+    param_estimates = pl.scan_parquet(Path(param_data_dir, "prod.parquet"))
     pmfs = get_pmfs(
         param_estimates=param_estimates,
         loc_abb=loc_abb,
         disease=disease,
-        right_truncation_required=fit_ed_visits,
+        right_truncation_required=False,
```

Comment:
See above on proposed argument change.
```suggestion
        right_truncation_required=right_truncation_required,
```